### DISCUSSION TITLE
Deployment on kubernetes ?

### DISCUSSION THREAD

**Discussion by ghost** [readysettech/readyset#4] [Q&A] [answered]
Hi ! 

Hi is there a way/workaround to deploy a ReadySet instance onto a cloud kubernetes cluster (e.g: gke) ? 

If not, do you happen to have some sort of kustomize/helm charts support in the near future ?

**Comment by alanarenda** [comment, marked as answer]
Hi @ValentinLevier! We do not currently have an easy way to deploy ReadySet onto GKE but providing Helm charts is a great idea. This will be a bit of an engineering lift but I will follow up here once they're ready. :)

**Comment by jseldess** [comment]
@ValentinLevier, just want to let you know that we know have a helm chart for deploying ReadySet into an Amazon EKS cluster. Here's the tutorial: https://docs.readyset.io/guides/deploy-readyset-kubernetes/. If you try it out, please share any issues you run into, or any feedback you have in general.

**Reply by ghost** [reply]
hi ! 
Thank you for your answer. Does it support Google GKE cluster too or is it exclusively for AWS EKS Cluster ?

**Reply by jbensal** [reply]
Hi Valentin!

We haven't officially tested the Helm chart with GKE yet. If you'd like to try it out, we'd be happy to support you and debug it to get it working!
